linera-core/src/client.rs

```diff
@@ -72,8 +72,6 @@ pub struct ChainClient<ValidatorNodeProvider, StorageClient> {
     /// Sequence number that we plan to use for the next block.
     /// We track this value outside local storage mainly for security reasons.
     next_block_height: BlockHeight,
-    /// Round number that we plan to use for the next block.
-    next_round: RoundNumber,
```

Comment:
As discussed we may want to keep this (and save it in wallet.json). Maybe create a task?

Reply:
I added https://github.com/linera-io/linera-protocol/issues/1041.

Yes, I don't think we will need this particular field. I imagine it will be the whole block proposal in the wallet.